# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/neofetch/hotfix-sparc.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- neofetch-7.1.0/neofetch.vanilla	2024-09-29 16:34:47.935987704 +0000
+++ neofetch-7.1.0/neofetch	2024-09-29 16:35:02.483987396 +0000
@@ -2123,7 +2123,7 @@
 
             # Get CPU cores.
             case $cpu_cores in
-                "logical" | "on") cores="$(grep -c "^processor" "$cpu_file")" ;;
+                "logical" | "on") cores="$(grep -c "^\(processor\|CPU.*online\)" "$cpu_file")" ;;
                 "physical") cores="$(awk '/^core id/&&!a[$0]++{++i} END {print i}' "$cpu_file")" ;;
             esac
         ;;
